.github/workflows/buildAndTest.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build And Test
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
```

Comment:
Semi-rhetorical question: should we at this point move to 11?

Reply:
Yes, for sure. It seems support for Java 8 is waning: https://www.oracle.com/java/technologies/java-se-support-roadmap.html

I'll set up a matrix build to run against java 8 and java 11 after this.